sdk/vision/azure-ai-vision-imageanalysis/README.md

```diff
@@ -0,0 +1,340 @@
+# Azure Image Analysis client library for Java
+
+The Image Analysis service provides AI algorithms for processing images and returning information about their content. In a single service call, you can extract one or more visual features from the image simultaneously, including getting a caption for the image, extracting text shown in the image (OCR) and detecting objects. For more information on the service and the supported visual features, see [Image Analysis overview][image_analysis_overview], and the [Concepts][image_analysis_concepts] page.
+
+Use the Image Analysis client library to:
+* Authenticate against the service
+* Set what features you would like to extract
+* Upload an image for analysis, or send an image URL
+* Get the analysis result
+
+[Product documentation][image_analysis_overview] 
+| [Samples][samples]
+| [Vision Studio][vision_studio]
+| [API reference documentation](https://learn.microsoft.com/java/api/com.azure.ai.vision.imageanalysis)
+| [Maven Package](https://mvnrepository.com/artifact/com.azure/azure-ai-vision-imageanalysis)
+| [SDK source code][sdk_source_code]
+
+## Getting started
+
+### Prerequisites
+
+*  [Java Development Kit (JDK)](https://learn.microsoft.com/azure/developer/java/fundamentals/java-jdk-install) with version 8 or above.
+* An [Azure subscription](https://azure.microsoft.com/free).
+* A [Computer Vision resource](https://portal.azure.com/#create/Microsoft.CognitiveServicesComputerVision) in your Azure subscription.
+  * You will need the key and endpoint from this resource to authenticate against the service.
+  * You can use the free pricing tier (`F0`) to try the service, and upgrade later to a paid tier for production.
+  * Note that in order to run Image Analysis with the `Caption` or `Dense Captions` features, the Azure resource needs to be from one of the following GPU-supported regions: `East US`, `France Central`, `Korea Central`, `North Europe`, `Southeast Asia`, `West Europe`, or `West US`.
+
+### Adding the package to your product
+
+[//]: # ({x-version-update-start;com.azure:azure-ai-vision-imageanalysis;current})
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-ai-vision-imageanalysis</artifactId>
+    <version>1.0.0-beta.1</version>
+</dependency>
+```
+[//]: # ({x-version-update-end})
+
+### Set environment variables
+
+To authenticate the `ImageAnalysisClient`, you will need the endpoint and key from your Azure Computer Vision resource in the [Azure Portal](https://portal.azure.com). The code snippet below assumes these values are stored in environment variables:
+
+* Set the environment variable `VISION_ENDPOINT` to the endpoint URL. It has the form `https://your-resource-name.cognitiveservices.azure.com`, where `your-resource-name` is your unique Azure Computer Vision resource name.
+
+* Set the environment variable `VISION_KEY` to the key. The key is a 32-character Hexadecimal number.
```

Comment:
To be clear on these two points, are these environment configurations specific to the samples below or something the SDK will respect in a production environment?

Reply:
@alzimmermsft good point, I added this sentence to make it clear: 

_Note that the client library does not directly read these environment variable at run time. The endpoint and key must be provided to the `ImageAnalysisClientBuilder` in your code. The code snippet below reads environment variables to promote the practice of not hard-coding secrets in your source code._